src/ActionModal/components/ActionModal.vue

```diff
@@ -556,9 +561,7 @@ export default {
   },
   methods: {
     updateEmoneyGasEstimate() {
-      if (this.network.id.startsWith(`emoney`)) {
-        this.gasEstimate = 200000
-      }
+      this.gasEstimate = 200000
```

Comment:
why not the tx separation here. but I guess we will have this from the API soon.

Reply:
ah yeah. True. Still good to have this already merged to know the specifics